src/SfxWeb/cypress/e2e/tree.cy.js

```diff
@@ -0,0 +1,110 @@
+import { addDefaultFixtures } from "./util.cy";
+
+context('tree', () => {
+    describe("accessibility", () => {
+        it("keyboard navigation", () => {
+            addDefaultFixtures();
+            cy.visit("");
+
+            //focused highlights tree
+            cy.get(".selected").focus();;
+            cy.get("[data-cy=tree-panel]").should("have.class", "focused").and("have.css", "border", "2px solid rgb(255, 255, 255)");
+            
+            //down arrow
+            cy.focused().type("{downarrow}");
+            cy.focused().contains("Applications")
+            cy.focused().should("have.attr", "tabindex", "0");
+            cy.get("[data-cy=tree]").within(() => {
+                cy.contains("Cluster").should("have.attr", "tabindex", "-1");
+            })
+            
+            //* expands all
+            cy.focused().type("*");
+            cy.get("[data-cy=tree]").within(() => {
+                cy.contains("Applications").within(() => {
```

Comment:
change this to a function which takes in the cy.contains since this logic is reused multiple times in this.